library/Cube/ProvidedHook/Cube/IcingaDbActions.php

```diff
@@ -19,7 +19,11 @@ public function createActionLinks(IcingaDbCube $cube)
 
         $paramsWithPrefix = [];
         foreach ($cube->getSlices() as $dimension => $slice) {
-            $paramsWithPrefix[$type . '.vars.' . $dimension] = $slice;
+            if (strpos($dimension, '.vars.') === false) {
```

Comment:
This leads to false positives if a custom var contains `.vars.`. Don't use strpos.